spring-boot-actuator/src/test/java/org/springframework/boot/actuate/endpoint/mvc/LogFileMvcEndpointTests.java

```diff
@@ -39,11 +38,15 @@
 /**
  * Tests for {@link LogFileMvcEndpoint}.
  *
- * @author Johannes Stelzer
+ * @author Johannes Edmeier
  * @author Phillip Webb
  */
 public class LogFileMvcEndpointTests {
 
+	/**
+	 * Comment necessary for checkstyle because this field is public - as it is with JUnit
```

Comment:
This caused me an error in eclipse... I'll remove it and See if it still Compiler...